common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1011,7 +1011,8 @@ def get_items(
             sort=[SORT_REVISION_FAVOR_DRAFT],
         )
 
-        modules = self._load_items(course_id, list(items))
```

Comment:
This change is necessary to enable loading the children of the graded content. Retrieval of problems takes about 14s with this change, and 2 minutes without it.